amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/base/BaseRMRestTest.java

```diff
@@ -957,4 +961,26 @@ protected boolean isRecordVersionInRecordFolder(FileModel testFile, RecordCatego
         }
     }
 
+    protected String getPreviousDate() {
+        Date date = new Date(System.currentTimeMillis());
+        Date previousDate = new Date(date.getTime() - 1);
+        // Conversion
+        SimpleDateFormat sdf= new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX");;
+        sdf.setTimeZone(TimeZone.getDefault());
+        return sdf.format(previousDate);
+    }
+
+    protected JSONObject editDispositionDateJson() {
```

Comment:
It would be good to at least have some Javadoc to mention this uses yesterday for the "as of" date.

Reply:
Corrected the naming convention and also added the description for the helper methods.